modules/meteroid/src/bin/server.rs

```diff
@@ -31,6 +31,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let store = meteroid_store::Store::new(
         config.database_url.clone(),
         config.secrets_crypt_key.clone(),
+        Arc::new(eventbus::memory::InMemory::new()),
```

Comment:
lets introduce some utility function for eventbus initialization which also adds handlers, this way we have one place where it is initialized and we will reuse that init function everywhere